src/engine/guidance/post_processing.cpp

```diff
@@ -308,22 +376,27 @@ void trimShortSegments(std::vector<RouteStep> &steps, LegGeometry &geometry)
     // Doing this step in post-processing provides a few challenges we cannot overcome.
     // The removal of an initial step imposes some copy overhead in the steps, moving all later
     // steps to the front.
-    // In addition, we cannot reduce the travel time that is accumulated at a different location.
-    // As a direct implication, we have to keep the time of the initial/final turns (which adds a
-    // few seconds of inaccuracy at both ends. This is acceptable, however, since the turn should
+    // In addition, we cannot reduce the travel time that is accumulated at a different
+    // location.
+    // As a direct implication, we have to keep the time of the initial/final turns (which adds
+    // a
+    // few seconds of inaccuracy at both ends. This is acceptable, however, since the turn
+    // should
```

Comment:
Can you re-wrap the lines, seems like clang-format breaks but does not join lines